packages/core-data/src/types/wp_template.ts

```diff
@@ -0,0 +1,81 @@
+/**
```

Comment:
Nit: Can we rename this to match the other file names in the repository? `wp-template.ts`, kebab case instead of snake?

Reply:
Good note, fixed